torch/nn/utils/parametrizations.py

```diff
@@ -90,7 +90,7 @@ def _power_method(self, weight_mat: torch.Tensor, n_power_iterations: int) -> No
             # This power iteration produces approximations of `u` and `v`.
             self._u = F.normalize(torch.mv(weight_mat, self._v),      # type: ignore[has-type]
                                   dim=0, eps=self.eps, out=self._u)   # type: ignore[has-type]
-            self._v = F.normalize(torch.mv(weight_mat.t(), self._u),  # type: ignore[has-type]
+            self._v = F.normalize(torch.mv(weight_mat.t(), self._u),
```

Comment:
Can you do the same for the line above?

Reply:
This was not flagged by `mypy` as unused so I kept it.